tutorials/fundamentals/create-element.md

```diff
@@ -0,0 +1,58 @@
+The `createElement()` function in JavaScript is used to programatically add elements to a web file.
+It takes one argument which is the type of element to create.
```

Comment:
`createElement()` has one required argument: the type of element to create, like `'div'` or `'img'`.